detekt.yml

```diff
@@ -647,6 +647,8 @@ datadog:
       - "java.net.URL.constructor(kotlin.String):java.net.MalformedURLException"
       - "java.io.PrintWriter.constructor(java.io.Writer):java.lang.NullPointerException"
       - "java.text.SimpleDateFormat.constructor(kotlin.String, java.util.Locale):java.lang.NullPointerException"
+      - "java.text.SimpleDateFormat.parse(kotlin.String):java.text.ParseException"
+      - "java.text.SimpleDateFormat.format(kotlin.Any):java.lang.IllegalArgumentException"
```

Comment:
Both can also throw an NPE

Reply:
Yup, I found it too now, it is somewhere hidden in the methods hierarchy but it is not mentioned in the main method description 🤦